libsql/src/database.rs

```diff
@@ -74,6 +74,7 @@ enum DbType {
         path: String,
         flags: OpenFlags,
         encryption_config: Option<EncryptionConfig>,
+        skip_saftey_assert: bool,
```

Comment:
Is this a typo? Title says `skip_safety_assert` but I see references to `skip_saftey_assert`.

Reply:
yeah this is def a typo, but its a private type so we can fix it without worry.